CHANGELOG

```diff
@@ -1,6 +1,9 @@
 v3.3.6 (XXXX-XX-XX)
 -------------------
 
+* fix issue #4582: ui query editor now has the same behavior as java driver while
```

Comment:
This is a bit too general. We should describe the new behavior.
e.g.: fix issue #4582: UI query editor now supports usage of empty string as bind parameter value